packages/integrations/src/extraerrordata.ts

```diff
@@ -75,27 +75,23 @@ export class ExtraErrorData implements Integration {
    * Extract extra information from the Error object
    */
   private _extractErrorData(error: ExtendedError): { [key: string]: unknown } | null {
-    let result = null;
     // We are trying to enhance already existing event, so no harm done if it won't succeed
     try {
       const nativeKeys = ['name', 'message', 'stack', 'line', 'column', 'fileName', 'lineNumber', 'columnNumber'];
-      const errorKeys = Object.getOwnPropertyNames(error).filter(key => nativeKeys.indexOf(key) === -1);
+      const errorKeys = Object.getOwnPropertyNames(error).filter(key => !nativeKeys.includes(key));
```

Comment:
Think we can’t use `.includes` because of IE, but might be taken care of when transpiled to the es5 bundle. @kamilogorek can you confirm?

Reply:
Exactly, we cannot. It's not transpiled, so we'd end up with requiring people to add polyfills for it.